src/coreclr/tools/aot/crossgen2/crossgen2_publish.csproj

```
@@ -7,8 +7,8 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <!-- Can't use NativeAOT in source build yet https://github.com/dotnet/runtime/issues/66859 -->
-    <NativeAotSupported Condition="'$(DotNetBuildFromSource)' == 'true'">false</NativeAotSupported>
+    <!-- Can't use NativeAOT in non-portable build yet https://github.com/dotnet/runtime/issues/66859 -->
+    <NativeAotSupported Condition="'$(PortableBuild)' == 'false'">false</NativeAotSupported>
```

PortableBuild should be treated as a true boolean in conditions. Runtime sets it to true as the default: https://github.com/dotnet/runtime/blob/135fec006e727a31763271984cd712f1659ccbd3/Directory.Build.props#L194

```suggestion
    <NativeAotSupported Condition="'$(PortableBuild)' != 'true'">false</NativeAotSupported>
```